app/views/site/search/_studies_list.html.erb

```diff
@@ -1,5 +1,5 @@
 <% @studies.each do |study| %>
-  <% study_description = strip_tags(study.description).truncate_words(100, omission: '... (continued)') %>
+  <% study_description = study.description.truncate_words(25, omission: '... (continued)') %>
```

Comment:
is there a reason this is getting switched from 100 to 25?

Reply:
That's actually a typo - good catch.